FirebaseAuth/Sources/Utilities/FIRAuthURLPresenter.m

```diff
@@ -162,8 +170,8 @@ - (void)finishPresentationWithURL:(nullable NSURL *)URL error:(nullable NSError
   _callbackMatcher = nil;
   id<FIRAuthUIDelegate> UIDelegate = _UIDelegate;
   _UIDelegate = nil;
-  FIRAuthURLPresentationCompletion completion = _completion;
-  _completion = nil;
+  FIRAuthURLPresentationCompletion completion = [_completion copy];
+  _completion = NULL;
```

Comment:
I think `nil` should be used in objc, no?

Reply:
`nil` and `NULL` will behave the same here, but NULL is more appropriate  because `_completion` as a reference to a block which is technically a C pointer. ([this article](https://nshipster.com/nil/) provides a summary for the difference)